samples/msal-node-samples/ElectronSystemBrowserTestApp/README.md

```diff
@@ -57,9 +58,18 @@ cd samples/msal-node-samples/standalone-samples/ElectronSystemBrowserTestApp
      - Select **Register** to create the application.
 1. In the list of pages for the app, select **Authentication**.
 1. Select **Add a platform**, select **Mobile and desktop applications**.
-1. In the **Redirect URIs** list, under **Suggested Redirect URIs for (mobile, desktop)** be sure to add `http://localhost:8080/` to the list of **Redirect URIs**. We will use the default `msal{Your_Application/Client_Id}://auth` **Redirect Uri** as our Custom URL Scheme, but more about that in the configure the application section.
+1. In the **Redirect URIs** list, under **Suggested Redirect URIs for (mobile, desktop)** be sure to add `http://localhost` to the list of **Redirect URIs**. We will use the default `msal{Your_Application/Client_Id}://auth` **Redirect Uri** as our Custom URL Scheme, but more about that in the configure the application section.
```

Comment:
This is confusing. The redirect URI is just the localhost. `msal{Your_Application/Client_Id}://auth` is used coincidentally for deep linking purposes, but it's not where the auth code response is received.

Reply:
Done